src/inline/plugin.py

```diff
@@ -293,6 +294,8 @@ class ExtractInlineTest(ast.NodeTransformer):
     arg_tag_str = "tag"
     arg_disabled_str = "disabled"
     arg_timeout_str = "timeout"
+    arg_devices_str = "devices"
+    diff_test_str = "diff_test"
```

Comment:
Can you take these (and other places related to `devices` and `diff_test`) out of this PR? They belong to the diff inline test functionality, so should be moved to a later PR instead.